default.json

```diff
@@ -21,6 +21,17 @@
       "matchPackagePatterns": ["*"],
       "enabled": false
     },
+    {
+      "matchManagers": [
+        "npm"
+      ],
+      "matchPackageNames": [
+        "@types/node"
```

Comment:
only for minor patch and pin versions

Reply:
Oops, sorry. Forgot to add my changes to the commit